### ISSUE TITLE
crypt() takes no keyword arguments

### ISSUE THREAD

**Issue by tumbl3w33d** [MindPointGroup/RHEL7-STIG#27]
Running the latest and greatest version of RHEL7-STIG against a freshly installed CentOS7 when the play executed the step
```
HIGH | RHEL-07-010490 | PATCH | Systems using Unified Extensible Firmware Interface (UEFI) must require authentication upon booting into single-user and maintenance modes
```
it resulted in the following error message:
```
FAILED! => {"failed": true, "msg": "[{u'regexp': u'^set superusers', u'line': u'set superusers=\"root\"'}, {u'regexp': u'^password_pbkdf2 root', u'line': u\"password_pbkdf2 root {{ rhel7stig_bootloader_password | grub2_hash(salt='KeokpkECTJeoDhEA5XtiLQ') }}\"}]: Unexpected templating type error occurred on (password_pbkdf2 root {{ rhel7stig_bootloader_password | grub2_hash(salt='KeokpkECTJeoDhEA5XtiLQ') }}): crypt() takes no keyword arguments"}
```
The cryptography module on the target system has version `2.0.3`.

Workaround: modify `filter_plugins/filters.py`
Turn `encrypted = crypt.crypt(password, salt=salt)` into `encrypted = crypt.crypt(password, salt)`

I didn't check the signature of `crypt` though, so I'm not sure that workaround makes sense, but at least you avoid the error and the system still works after a reboot.

**Comment by samdoran** [comment]
Thanks for reporting this. I'll look into it shortly.

**Comment by samdoran** [comment]
I have an open PR #23 that should fix this since it's no longer using `crypt`. It uses `passbilb` instead. Can you try that and see if it resolves your issue?

**Comment by tumbl3w33d** [comment]
Yeah, gonna try that tomorrow at work.

**Comment by tumbl3w33d** [comment]
@samdoran I can confirm your PR works and my problem is solved as well. By the way, my cryptography lib obviously wasn't the default CentOS7 version (which is 1.3.x). It must have been upgraded when installing ansible on that machine and probably this problem arises then when you run 2.x.
I tested your PR with a (really) fresh CentOS7 installation and it works. :)

You might want to add a local passlib installation to your readme as dependency though. I first got the error that it's required and installed it on the remote machine, which didn't fix it. Only after I installed passlib locally I was able to get through.

Feel free to close this here and thank you for the fast response!
